src/iosrv/init.c

```diff
@@ -946,15 +949,61 @@ main(int argc, char **argv)
 			break;
 		}
 
-		/* use this iosrv main thread's context to dump Argobot internal
-		 * infos upon SIGUSR1
+		/* open specific file to dump ABT infos and ULTs stacks */
+		if ((sig == SIGUSR1 || sig == SIGUSR2) && abt_infos == NULL) {
+			abt_infos = fopen("/tmp/ABT_dumps.txt", "a");
```

Comment:
maybe something like /tmp/daos_dump_20200724_09_20.txt.

Reply:
Yes, we can use the date in file name to indicate 1st SIGUSR1/SIGUSR2 trigger, but don't forget that, due to asynchronous dump mode (executed in an other Argobots context), we will keep the same file opened/used during full server life, their will be internal timestamps to delimit consecutive dumps occurrences. If its size becomes too huge, it may become necessary to copy/truncate it from external/admin.